packages/design-system/src/utils/useMouseDownOutsideRef.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useRef, useCallback } from '@web-stories-wp/react';
+/**
+ * Internal dependencies
+ */
+import { noop } from './noop';
+
+function useMouseDownOutsideRef(onPointerDownOutside = noop) {
```

Comment:
💭 Let's add a little docstring description here to describe what this does. Specifically that it returns a ref 😄

Reply:
ope. woops. thanks for pointing out